tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -1,5 +1,177 @@
 [assembly: System.Reflection.AssemblyMetadata("RepositoryUrl", "https://github.com/DataDog/dd-trace-dotnet.git")]
 
+namespace Datadog.Trace
+{
+    public sealed class AgentProcessManagerLoader
+    {
+        public AgentProcessManagerLoader() { }
+    }
+    public static class CorrelationIdentifier
```

Comment:
Um.... :wat:

I guess we need to sort the namespaces in here or something 😬

Reply:
I didn't do anything, just update the snapshot I guess it is because the sorting indeed.